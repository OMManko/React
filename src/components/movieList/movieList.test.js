import { shallow } from 'enzyme';
import React from 'react';
import MovieList from './movieList';

describe('<MovieList />', () => {
    it('should render movie list and match snapshot', () => {
        const component = shallow(<MovieList/>);
        expect(component).toMatchSnapshot();
    });
});
