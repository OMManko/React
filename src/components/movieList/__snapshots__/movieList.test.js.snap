// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieList /> should render movie list and match snapshot 1`] = `ShallowWrapper {}`;
